### PR TITLE
fuzz: Fix the building errors with recent changes

### PR DESCRIPTION
**Pull request by likebreath**
This patch adds two required dependencies to fuzz/Cargo.toml, and fixes
the building error on the 'block' fuzzer.

Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

File: fuzz/Cargo.toml [hunk 1]
```diff
@@ -17,12 +17,14 @@ virtio-devices = { path = "../virtio-devices" }
 vmm-sys-util = ">=0.3.1"
 vm-virtio = { path = "../vm-virtio" }
 vm-memory = "0.2.1"
+seccomp = { git = "https://github.com/firecracker-microvm/firecracker", tag = "v0.22.0" }
```
Comment (likebreath) [inline fuzz/Cargo.toml:20]: @rbradford I am not sure about the use of `[dependencies]` and `[patch.crates-io]`. Naively, I put the direct dependency as `[dependencies]` and  the in-direct dependency as `[patch.crates-io]`. Any inputs?

**Comment by likebreath** [comment]
BTW, I also added a GH action to test the cargo fuzz build [here](https://github.com/likebreath/cloud-hypervisor/pull/1). Please let me know if we want it in our repo here.

File: fuzz/Cargo.toml (hunk 1 above)
Comment (rbradford) [reply fuzz/Cargo.toml:20]: Both of these should be in dependencies. The patch is if we need to override a dependency from a crate we depend upon.

Try and keep these lists lexically sorted fixing any that are wonky.

Adding the action sounds like a good idea!

Comment (likebreath) [reply fuzz/Cargo.toml]: @rbradford Fixed. Thanks for the explanation. I will add the GH action with a different PR.

**Comment by likebreath** [comment]
@michael2012z @MrXinWang Could you please take a look at the aarch64 CI failure? Thanks a lot.

```
Cloning repository https://github.com/cloud-hypervisor/cloud-hypervisor.git
ERROR: Failed to clean the workspace
jenkins.util.io.CompositeIOException: Unable to delete '/var/lib/jenkins/workspace/cloud-hypervisor_PR-1778'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts. (Discarded 16 additional exceptions)
```

**Comment by MrXinWang** [comment]
@likebreath I've manually deleted the folder? Could you please re-trigger the CI again to see how it goes (I don't have the privilege to do that)? Thanks!

**Comment by likebreath** [comment]
@MrXinWang Thanks. It is now working. :)

**Review by sboeuf — APPROVED** [review]

**Review by rbradford — APPROVED** [review]

**Pull request merged**